locale/bg.po

```diff
@@ -5473,6 +5478,43 @@ msgstr ""
 "\n"
 "Натиснете бутона за извършване на самоубийство, за да пропуснете този процес, но не забравяйте, че винаги има шанс да се спасите!"
 
+#, fuzzy
+#~ msgid "PASSIVE_REPRODUCTION_PROGRESS_EXPLANATION"
```

Comment:
Did you manually merge the translations or where have these changes come from?

Reply:
This doesn't look like that gettext should do. So can I ask you to, first make a backup of your changes to en.po, then copy the files from master and replace what you have now, run the localization update and then put back in just the one English translation.

Reply:
I checked out only the locale files from master, ran `update_localization.rb`, then added back my change. I used poedit.  

I'll try again manually copying them now.

Reply:
I hope it's correct now. I'm still a bit confused, to be honest.

Reply:
Most of the files look fine now, but for some reason new fuzzy translations have been added for PASSIVE_REPRODUCTION_PROGRESS for multiple languages. I don't think gettext should add such translations other than when initially adding the text.

Did you do the following:
- Checkout master somewhere separately
- Copy paste all the .po files to your branch
- Run update localization
- Edit just en.po to put in that one needed translations
- Commit

That should ensure that the translation changes are pristine, gettext doesn't cause any intermediate changes or new guessed translations. If any not really well done merges or intermediate changes are present when updating localizations, gettext will do stuff that shouldn't be done.
If random changes exist after those steps I outlined above, we'll just live with them as gettext does that, but looking at the diff it seems to me that you may have done something that prompted gettext to do unnecessary changes.